guides/v2.1/cloud/cdn/fastly-vcl-badreferer.md

```diff
@@ -11,57 +11,104 @@ functional_areas:
+
+    -  Click **Upload** to attach the dictionary to your Fastly service configuration.
+
+1.  Add the list of domain names to block to the the `referrer_blocklist` dictionary:
+
+    -  Click the Settings icon for the `referrer_blocklist` dictonary.
+
+    -  Add and save key-value pairs in the new dictionary. For this example, each **Key** is a website referrer URL that you want to block and the **Value** is `true`. 
+       
+       ![Add bad referrer dictionary items]
+	 
+    -  Click **Cancel** to return to the system configuration page.
 
 For more information on using Edge Dictionaries with your VCL snippets, see Fastly's [Creating and using Edge Dictionaries](https://docs.fastly.com/guides/edge-dictionaries/creating-and-using-dictionaries) and their example [custom VCL snippets](https://docs.fastly.com/guides/edge-dictionaries/creating-and-using-dictionaries#custom-vcl-examples).
 
-## Create badreferer.json {#vcl}
+## Create a custom VCL snippet to block referrer spam
 
-For this example, you may only want to run it against the Production server. You can also add it to Staging for testing.
+The following custom VCL snippet code (JSON format) checks if incoming requests are from a referrer site included in the `referrer_blocklist` edge dictionary. If the site is in the block list, the request is blocked.
 
-Create an `badreferer.json` file with the following JSON content:
 
 ```json
 {
-  "name": "badreferer",
+  "name": "block_bad_referrer",
   "dynamic": "0",
   "type": "recv",
   "priority": "5",
   "content": "set req.http.Referer-Host = regsub(req.http.Referer, \"^https?://?([^:/\\s]+).*$\", \"\\1\"); if (table.lookup(referer_blocklist, req.http.Referer-Host)) { error 403 \"Forbidden\"; }"
 }
 ```
+Review the code to determine if you need to change any values:
+
+  -  `name`: Name for the VCL snippet. For this example, we used the name `block_bad_referrer`.
+  
+  -  `dynamic`: Value 0 indicates that this is a [regular snippet](https://docs.fastly.com/guides/vcl-snippets/using-regular-vcl-snippets)  that you upload to the versioned VCL for the Fastly configuration.
+
+  -  `priority`: Determines when the VCL snippet runs. The priority is set to 5, which means that this snippet code runs before any of the default Magento VCL snippets (magentomodule) that have a priority of 50.
+
+  -  `type`: This VCL is a `recv` snippet type which adds the snippet code to the `vcl_recv`subroutine below the boilerplate VCL and above any objects.
+ 
+  -  `content`: When the VCL code in this example runs, it captures the host of a referer website into a header, and then compares the host name to the list of URLs in the `referrer_blocklist` dictionary. If the host name matches, the request is blocked with a `403 Forbidden` error. See the [Fastly VCL reference](https://docs.fastly.com/vcl/reference/) for information about creating Fastly VCL code snippets.
```

Comment:
Referrer is spelled differently here. Bringing attention to it in case you don't want it like this.